src/Init/Data/Nat/Basic.lean

```diff
@@ -629,7 +629,7 @@ protected theorem sub_lt_sub_left : ∀ {k m n : Nat}, k < m → k < n → m - n
 @[simp] protected theorem zero_sub (n : Nat) : 0 - n = 0 := by
   induction n with
   | zero => rfl
-  | succ n ih => simp [ih, Nat.sub_succ]
+  | succ n ih => simp [ih, Nat.sub_succ]; decide
```

Comment:
Could you please use `(config := ...)` here.

Reply:
This one failed with:

```
Init/SimpLemmas.lean:156:84: error: unknown constant 'Lean.Meta.Simp.Config'
```

Any suggestions/workarounds?

Reply:
I replaced `simp` by `simp only`